caas/kubernetes/provider/constraints.go

```diff
@@ -8,6 +8,8 @@ import (
 	"github.com/juju/juju/environs/context"
 )
 
+const CAASProviderType = "kubernetes"
```

Comment:
we have providerType const in caas/kubernetes/provider/init.go
just need to export that

Reply:
`caas/kubernetes/provider/init.go` imports `CAASProviderType` from here.